portal/models/questionnaire_bank.py

```diff
@@ -256,7 +256,8 @@ def validate_classification_count(qbs):
 
     @staticmethod
     def most_current_qb(user, as_of_date=None):
-        """Return "most current" qb for user, or None
+        """Return tuple of "most current" qb for user (or None), and recurrence
```

Comment:
Silly documentation style, but we'll try and stick to it.

First line should be < 80 chars, just a one liner.  If longer description is needed, it should be in its own paragraph:

```
"""Return two item tuple (qb, iteration_count) for user

Returns tuple of "most current" qb for user (or none if N/A) and the recurrence iteration number if the found QB is recurring, else None.
"""
```